cli/src/commands/extensions/api.rs

```diff
@@ -91,6 +91,19 @@ struct ProcessException {
     strict: bool,
 }
 
+#[cfg(unix)]
+impl From<ProcessException> for permissions::Permissions {
+    fn from(process_exception: ProcessException) -> Self {
+        Self {
+            read: process_exception.read,
+            write: process_exception.write,
+            run: process_exception.run,
+            env: Permission::Boolean(false), // TODO: replace with inherit
```

Comment:
If that's ok, since we went with no inherit, I'd switch this to `true` so it mimicks the current behavior (i.e. allow all) and we can tackle this in the other PR.